common/client_config.go

```diff
@@ -0,0 +1,35 @@
+package common
+
+import (
+	"fmt"
+	"time"
+
+	clients_v2 "github.com/Layr-Labs/eigenda/api/clients/v2"
+)
+
+// ClientConfigV2 contains all non-sensitive configuration to construct V2 clients
+type ClientConfigV2 struct {
```

Comment:
```suggestion
type ClientsV2Config struct {
```

Reply:
does this make more sense?

Reply:
Also didn't we want to separate retriever clients from disperser clients so that we can start proxy in read-only mode?

Reply:
> does this make more sense?

I don't feel strongly about this- my tendency would be to leave it as-is to avoid refactor work. If you feel strongly, I can make the change, LMK

Reply:
> Also didn't we want to separate retriever clients from disperser clients so that we can start proxy in read-only mode?

I think this might be a change to consider in the future. The problem with making this change is that I think that train of logic would lead us to abandon the aggregate `ProxyConfig` struct entirely: 

e.g. "what if the user wants a proxy which only reads from relays, and not validator nodes? Perhaps we shouldn't group those configs together either?"

I think this might actually be the right decision in the future, but it would be a large change.